django/db/models/expressions.py

```diff
@@ -719,7 +719,8 @@ def get_source_fields(self):
     def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize=False, for_save=False):
         c = self.copy()
         c.is_summary = summarize
-        c.condition = c.condition.resolve_expression(query, allow_joins, reuse, summarize, False)
+        if hasattr(c.condition,"resolve_expression"):
```

Comment:
The flake8 error is due to missing a space after the comma here. Please use single quotes instead of double.